label_studio/annotation_templates/generative-ai/visual-ranker/config.yml

```
@@ -0,0 +1,61 @@
+title: "Visual Ranker"
+type: community
+group: Generative AI
+order: 5
+image: /static/templates/visual-ranker.png
+details: Here is a demo that showcases how to construct an interface enabling users to drag, arrange, and rank responses from different Large Language Models (LLMs) and chatbot systems based on their relevance to a specified prompt and their perceived bias, thus facilitating an intuitive and visual understanding of how well these AI systems align with the topic and the degree of subjective interpretation they exhibit.
+config: |
+  <View>
+        <Style>
+          .product-panel {
+            display: flex;
+            align-items: center;
+            box-shadow: 0px 5px 15px rgba(0,0,0,0.1);
+            padding: 20px;
+            border-radius: 10px;
+            background-color: #f8f8f8;
+            font-size: 1.3em;
+            transition: transform .2s; 
+            color: #333;
+          }
+        
+          .product-panel:hover {
+            transform: scale(1.01);
+          }
+          .htx-ranker-item { padding: 0; border: 0; margin: 0 auto;}
```

```suggestion
          .htx-ranker-item { padding: 0; border: 0; align-self: center; }
```
that fixes the problem with images jumping

---

```suggestion
          .htx-ranker-item { padding: 0; border: 0; align-self: center; }
          .htx-ranker-item img { width: 100%; }
```
And even better — to display any image in a correct size